src/SDKs/CognitiveServices/dataPlane/Vision/ComputerVision/ComputerVision/Microsoft.Azure.CognitiveServices.Vision.ComputerVision.csproj

```diff
@@ -6,10 +6,10 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.CognitiveServices.Vision.ComputerVision</PackageId>
     <Description>This client library provides access to the Microsoft Cognitive Services ComputerVision APIs.</Description>
-    <VersionPrefix>1.0.2-preview</VersionPrefix>
+    <VersionPrefix>1.0.3</VersionPrefix>
```

Comment:
Please confirm you plan on releasing a stable version here. If yes, versioning should start from `1.1.0`
Also, all the projects need an `AssemblyInfo.cs` file similar to [this](https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/Compute/Management.Compute/Properties/AssemblyInfo.cs) with appropriate package versions.